### ISSUE TITLE
Remove `.python-version`

### ISSUE THREAD

**Issue by FichteFoll** [naokazuterada/MarkdownTOC#158]
On ST4 builds, the plugin is ignored entirely because the `.python-version` file is interpreted and only supports 3.3 and 3.8 for the two available runtimes in that version. The exact exception is as follows:

```
ignoring plugins in package MarkdownTOC, version file Packages/MarkdownTOC/.python-version contains an invalid version: 3.7.1\x0a
```

The file was added in https://github.com/naokazuterada/MarkdownTOC/commit/f97f221dc267dfed2dba1e7cacea4253031f30f7, but I don't see a particular reason for it.

**Comment by naokazuterada** [comment]
This file is for using "black" to make code styling common. 
I'd like to keep the version number. Will it work if I set it to `3.8.3`? If you tell us the version that works, we will change it to that value. Or could you create a PR with the changed version?

**Comment by FichteFoll** [comment]
I know that `3.8` works, if that's fine for you. The version ST4 bundles is `3.8.3` currently and will likely increase, but I don't see much of a point in pinning the patch version since that will likely be updated in the future.

**Comment by naokazuterada** [comment]
@FichteFoll @nickhstr please check working correctly.

**Comment by FichteFoll** [comment]
Should work like that.

**Comment by naokazuterada** [comment]
Thank you for all!

Released [4.1.1](https://github.com/naokazuterada/MarkdownTOC/releases/tag/4.1.1).
